datalad/support/sshconnector.py

```diff
@@ -123,7 +138,16 @@ def __call__(self, cmd):
 
         # TODO: pass expect parameters from above?
         # Hard to explain to toplevel users ... So for now, just set True
```

Comment:
May be move the old comment to line 146 (right before `runner.run(...`), since this is, what it is referring to.